# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/fnord/hotfix-buffer.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- fnord-1.11.orig/buffer_stubborn.c	2001-11-24 19:52:40.000000000 +0100
+++ fnord-1.11/buffer_stubborn.c	2026-07-26 06:30:45.742141112 +0200
@@ -1,7 +1,7 @@
 #include <errno.h>
 #include "buffer.h"
 
-int buffer_stubborn(int (*op)(),int fd,const char* buf, unsigned int len) {
+int buffer_stubborn(int (*op)(int,const char*,unsigned int),int fd,const char* buf, unsigned int len) {
   int w;
   while (len) {
     if ((w=op(fd,buf,len))<0) {
